src/quo/components/bottom_sheet/style.cljs

```diff
@@ -23,12 +25,13 @@
    :right            0
    :bottom           0})
 
-(defn content-container
-  [window-height]
-  {:background-color        (:ui-background @colors/theme)
-   :border-top-left-radius  border-radius
-   :border-top-right-radius border-radius
-   :height                  (* window-height 2)})
+(defn content-container  [window-height]
+  (let [dark? (theme/dark?)]
+    {:background-color (if dark? colors2/neutral-90 colors/white)
```

Comment:
hi @chrispader 
1. we should not change `quo` files
2. please use colors2 only. Here [themes-colors](https://github.com/status-im/status-mobile/blob/180df967e81c9c417e71db19670d1c64b56b2cfb/src/quo2/foundations/colors.cljs#L189) function can be used.
3. Also, if possible please do implementation and integration of component in separate PRs (easy to review)

Reply:
Yes, I think we can copy this component in quo2 (or in ui2?) and make changes.
cc: @flexsurfer

Reply:
yeah but not in this PR